package.json

```diff
@@ -20,7 +20,7 @@
     "lint:syncpack": "syncpack lint",
     "lint:turbo": "turbo lint",
     "release": "turbo build && pnpm changeset publish",
-    "test:turbo": "turbo test"
+    "test:turbo": "turbo run test --filter='!@liam-hq/e2e'"
```

Comment:
If you do this, it might be easier to change the e2e test run command like `pnpm test-e2e`

Reply:
I agree🙏.I've changed the command.https://github.com/liam-hq/liam/pull/660/commits/b106322313d30b6eb1762dd2713bd0c8fe92525d

Reply:
fixed: https://github.com/liam-hq/liam/pull/660/commits/8b8728829335af1e2dc323d00232b2446df09975

We can run ``URL=localhost:5173 pnpm test:e2e``